comparators/added_place.js

```diff
@@ -3,8 +3,9 @@
 module.exports = addedPlace;
 
 function addedPlace(newVersion, oldVersion) {
+  var result = false;
```

Comment:
This is very much a matter of taste so ignore it if you feel otherwise :)

I think for primitive values as `false` it is not a code duplication to` return false`.
You probably refactored to `var result = false; return result;` to make it easier to refactor in the future
but imho I think it makes the code harder to read since you need to know what result is.

tldr: I would suggest to `return false` instead `var result = false; return result;` everywhere.

![giphy 8](https://cloud.githubusercontent.com/assets/1288339/24865269/5fccbcbc-1dcc-11e7-8c1a-0620878112d4.gif)